### PR TITLE
LITE-20272 Add new capabilities and examples to connect-cli

### PR DESCRIPTION
**Pull request by vlas**

### CODE REVIEW

File: tests/plugins/project/test_extension_helpers.py [hunk 1]
```diff
@@ -85,8 +91,166 @@ def test_bootstrap_extension_project(
...
+        'capabilities': {
+            'asset_purchase_request_processing': [
+                'draft',
+                'scheduled',
+                'revoking',
+                'revoked',
+            ],
+        },
+    }
+    mocked_open = mocker.patch(
```
Comment (gab832) [inline tests/plugins/project/test_extension_helpers.py:245]: Typo maybe with the mocked_open variable used multiple times?

**Review by gab832 — APPROVED** [review]
Nice job!

**Review by ffaraoneim — CHANGES_REQUESTED** [review]

File: connect/cli/core/config.py [hunk 2]
```diff
@@ -38,6 +38,14 @@ def add_account(self, id, name, api_key, endpoint=DEFAULT_ENDPOINT):
     def active(self):
         return self._active
 
+    @property
+    def is_active_vendor(self):
```
[the remark this answers is not in the document]
Comment (vlas) [reply connect/cli/core/config.py]: thanks, changed this

File: connect/cli/plugins/project/extension_helpers.py [hunk 3]
```diff
@@ -245,6 +383,22 @@ def _check_statuses(capability, stat, errors):
...
+            errors.append(f'Status `{stat}` on capability `{capability}` is not allowed.')
+    elif capability == 'tier_account_update_request_processing':
+        if stat not in TIER_ACCOUNT_UPDATE_STATUSES:
+            errors.append(f'Status `{stat}` on capability `{capability}` is not allowed.')
+    elif capability == 'part_usage_file_request_processing':
+        if stat not in CHUNK_FILE_STATUSES:
+            errors.append(f'Status `{stat}` on capability `{capability}` is not allowed.')
+    elif capability == 'usage_file_request_processing':
+        if stat not in USAGE_FILE_STATUSES:
+            errors.append(f'Status `{stat}` on capability `{capability}` is not allowed.')
```
Comment (marcserrat) [inline connect/cli/plugins/project/extension_helpers.py:401]: Can you please simplify / extract?
https://sonarcloud.io/project/issues?id=connect-cli&open=AXzmfRmI9W6MU9yIVBwu&pullRequest=72&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL

Comment (vlas) [reply connect/cli/plugins/project/extension_helpers.py:397]: should be fixed, thanks

**Review by ffaraoneim — APPROVED** [review]

**Pull request merged**